lib/developer_portal/lib/liquid/drops/service.rb

```diff
@@ -181,7 +181,7 @@ def metrics
         Drops::Metric.wrap(@service.metrics.top_level)
       end
 
-      # TODO: remove this and remove all references, views, controllers
+      # NOTE: keeping in case somebody already using 'admin_support_email' liquid
```

Comment:
Then it should return nil and I think we should think about removing for real those deprecated tags.
@secretsurf @3scale/product @3scale/support 
We need to make a plan on when and how to remove those.

My guess would be in 3 steps (3 releases)

1. add a logger there to know if people are still using it. And collect data.
2. Send a deprecation notice to those tenants
3. Remove the method definitively

Reply:
@hallelujah @macejmic 
Should we add here as well 1 for the tech_support_email?
```ruby
hidden
deprecated "Use **support_email** instead."
def tech_support_email
  @service.support_email
end
```

Reply:
@hallelujah I would be in favour of add a logger so we can collect usage data and go from there.

Reply:
So I guess this should be like the following example (please confirm @hallelujah ):
```ruby
# NOTE: keeping in case somebody already using 'admin_support_email' liquid
hidden
deprecated "Use **support_email** instead."
def admin_support_email
  Rails.logger.info "Liquid::Drops::Service#admin_support_email is being used by Service ##{@service.id}"
  nil
end
```

Reply:
@josemigallas No need to add the `tech_support_email` here 😄 If it isn't there is because nobody is using it which is what we want 😄

Reply:
👍  Sorry this method do not have to change, forget about my first comment.